pkg/pull/peek.go

```diff
@@ -0,0 +1,57 @@
+package pull
+
+import (
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/kots/pkg/logger"
+	"github.com/replicatedhq/kots/pkg/upstream"
+)
+
+type PeekOptions struct {
+	HelmRepoURI   string
+	Namespace     string
+	LocalPath     string
+	LicenseFile   string
+	CurrentCursor string
+	Silent        bool
+}
+
+// Peek will retrieve all later versions of the application specified in upstreamURI
+// using the options specified in peekOptions. It returns a list of versions.
+func Peek(upstreamURI string, peekOptions PeekOptions) ([]upstream.Update, error) {
```

Comment:
Why is this `Peek` and not `GetUpdates`?  This needs a better name or a revised comment about what it does.